Automattic/newspack-custom-content-migrator
# feat(EmbarcaderoMigrator): allow only a set of categories

## How to test
- Run the new command `newspack-content-migrator embarcadero-rearrange-categories`
- Notice the categories set (all the categories should belong to the list from the constant `ALLOWED_CATEGORIES`, or the `General` category).

---
